js/help.js

```diff
@@ -0,0 +1,49 @@
+const helpOverlay = new Image();
+helpOverlay.src = 'images/help-image.png';
+
+const helpBtn = document.querySelector('.help');
+
+let helpImage = false;
+
+function drawHelp() {
+  if (helpImage === true) {
+    ctx.drawImage(helpOverlay, 15, 0);
+  }
+}
+
+function introText() {
```

Comment:
As I've mentioned, better to name function as a verb, like `drawIntroText()` or something like that.